client/src/components/Events/Events.jsx

```diff
@@ -0,0 +1,193 @@
+
+const Events = () => {
+  const [currState, setCurrState] = useState(0);
+  const [index1, setIndex1] = useState(1);
+  const [index2, setIndex2] = useState(2);
+  const [index3, setIndex3] = useState(3);
+  const [index4, setIndex4] = useState(4);
+  const [color, setColor] = useState(0);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      if (currState === 7) setCurrState(0);
+      else setCurrState(currState + 1);
+      setColor((currState + 1) % 8);
+      // setIndex1((((currState + 1) % 8) + 1) % 8);
+      // setIndex2((index1 + 1) % 8);
+      // setIndex3((index2 + 1) % 8);
+      // setIndex4((index3 + 1) % 8);
+      setIndex1((((currState + 1) % 8) + 1) % 8);
+      setIndex2((((((currState + 1) % 8) + 1) % 8) + 1) % 8);
+      setIndex3((((((((currState + 1) % 8) + 1) % 8) + 1) % 8) + 1) % 8);
+      setIndex4(
+        (((((((((currState + 1) % 8) + 1) % 8) + 1) % 8) + 1) % 8) + 1) % 8
+      );
+    }, 8000);
+    return () => clearTimeout(timer);
+  });
+  const goToNext = (currState) => {
+    setCurrState(currState);
+    setColor(currState);
+    setIndex1((currState + 1) % 8);
+    setIndex2((((currState + 1) % 8) + 1) % 8);
+    setIndex3((((((currState + 1) % 8) + 1) % 8) + 1) % 8);
+    setIndex4((((((((currState + 1) % 8) + 1) % 8) + 1) % 8) + 1) % 8);
+  };
+
+  return (
+    <div className="bg-custom-dark" id="container-style">
+      <div id="description" className="bg-custom-dark sm:pb-6">
+        <div className="bg-custom-dark">
+          <h1
+            id="heading"
+            className="text-gray-300 text-center text-2xl pb-4 font-bold pt-12"
+          >
+            OUR EVENTS
+          </h1>
+        </div>
+        <div className="lg:flex lg:flex-row sm:flex sm:flex-col justify-center items-center xl:mx-44 lg:mx-20 sm:mx-10">
+          <div
+            id="photo-detail"
+            className="pt-10 lg:pl-10 pb-10 lg:w-1/2 sm:flex sm:flex-col sm:justify-center sm:items-center lg:block"
+          >
+            <div className="lg:pr-10">
+              <h1
+                id="photo-title"
+                className="sm:text-4xl lg:text-5xl text-gray-400 font-bold mb-8"
+              >
+                {eventsData[currState].title}
+              </h1>
+            </div>
+            <p
+              id="photo-body"
+              className=" text-gray-300 sm:px-10 md:px-20 lg:px-0 lg:pr-10 sm:text-center lg:text-left"
+            >
+              {eventsData[currState].body}
+            </p>
+            {/* <ArrowRightAltIcon className="ml-1" /> */}
+            <button className="bg-gray-300 mt-10 p-2 pl-3 w-40 text-left rounded-sm font-semibold flex justify-left items-center">
+              READ MORE
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                class="w-6 h-6"
```

Comment:
why two classnames?? 😞  And attributes are camel-cased in jsx.

Reply:
Removed the class attribute... a903394